src/Database/Table/Selection.php

```diff
@@ -350,6 +350,18 @@ public function order($columns)
 
 
 	/**
+	 * Resets current order clause.
+	 * @return self
+	 */
+	public function resetOrder()
+	{
+		$this->emptyResultSet();
+		call_user_func_array(array($this->sqlBuilder, 'setOrder'), [[], []]);
```

Comment:
no need for call_user_func_, $this->sqlBuilder->setOrder([], []); is sufficient.